contrib/docker-compose-for-tests.yml

```diff
@@ -32,24 +32,28 @@ services:
       - "16390:6390"
 
   redis-2-8:
+    platform: linux/amd64
```

Comment:
What are these all about?

Reply:
Can we revert this? let's keep the docker-compose file platform-agnostic.

Reply:
Ah yes, I am running the test on M1 Apple silicone machine. There are some warnings shown and that's why I put it there. Sure, I will remove it

Reply:
Thanks!